modules/Rock.cmake

```diff
@@ -1446,6 +1488,11 @@ function(rock_test_common TARGET_NAME)
         set(TARGET_NAME "${PROJECT_NAME}-test")
     endif()
 
+    option(ROCK_TEST_CXX_GCOVR_GENERATION_ENABLED "Generate gcovr reports after test runs" OFF)
+    if (ROCK_TEST_CXX_GCOVR_GENERATION_ENABLED)
+        rock_cxx_coverage_report(${PROJECT_NAME})
+    endif()
+
```

Comment:
This can't be called here as it is global (per-project) and not per-target. Call it right before the testing bits in rock_standard_layout